charts/hivemq-platform/values.yaml

```diff
@@ -88,17 +88,41 @@ restApi:
   # Enables or disables authentication and authorization.
   authEnabled: false
 
-# Configures the HiveMQ License information.
+# Configures the HiveMQ Platform license information.
 license:
   # Creates a default HiveMQ Platform license as a Kubernetes Secret.
   create: false
-  # The name of a secret with the license information.
+  # The name of a Kubernetes Secret with the HiveMQ Platform license information.
   # To create a new secret with a license, set create=true. Otherwise, false to reuse an existing one.
   name: ""
-  # Inlines the License information as an encoded 64-bytes string.
+  # Inlines the HiveMQ broker license information as an encoded 64-bytes string.
   data: ""
-  # Overrides the License information via file using --set-file license.lic.
+  # Overrides the HiveMQ broker license information via file using --set-file license.lic.
   overrideLicense: ""
+  # Additional HiveMQ broker licenses to include as .lic files of the same Kubernetes Secret license.
+  additionalBroker: {}
+    ## Defines the name of the HiveMQ broker license in the Kubernetes secret (the .lic suffix will be added automatically).
+    #<broker-license-name>:
+    #  # Inlines the HiveMQ broker license information as an encoded 64-bytes string.
+    #  data: ""
+    #  # Overrides the HiveMQ broker license information via file using --set-file license.lic.
+    #  overrideLicense: ""
+  # HiveMQ Enterprise Extension licenses to include as part of the same Kubernetes Secret license.
+  extensions: {}
+    ## Defines the name of the HiveMQ Enterprise Extension license in the Kubernetes secret (the .elic suffix will be added automatically).
+    #<extension-license-name>:
```

Comment:
```suggestion
    #extension-license-name:
```

Reply:
same comment as the `license-name`. Let me know what you think